client/src/components/UI/Button.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import classes from './Button.module.css'
+
+declare interface ButtonProps {
+  children?: string| string[];
+  type?: "submit" | "reset" | "button";
+  classText: string;
```

Comment:
whats the purpose of class text?

Reply:
I'm passing different css classes for different buttons as props. I've changed the name of this prop to className.